src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/DocumentedCommand.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.CommandLine;
+
+namespace NuGet.CommandLine.XPlat.Commands
+{
+    public class DocumentedCommand : CliCommand
+    {
+        public DocumentedCommand(string name, string description, string helpUrl)
```

Comment:
`CliCommand` makes the description optional. My gut feeling is that we should add a description to every command, so I thought it'd be fine to make it non-optional for this wrapper class. Well, I guess you could just explicitly pass in a null.

But if anyone thinks that we should follow the `CliCommand` pattern and make `string description = null`, then I should re-order these parameters.

Any thoughts?